modules/exploits/windows/ftp/ftpshell_cli_bof.rb

```diff
@@ -0,0 +1,107 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::TcpServer
+  include Msf::Exploit::Seh
+  include Msf::Exploit::Remote::Egghunter
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'FTPShell client 6.70 (Enterprise edition) Stack Buffer Overflow',
+      'Description'    => %q{
+          This module exploits a buffer overflow in the FTPShell client 6.70 (Enterprise edition) allowing remote
```

Comment:
Line wrap at ~80 columns is preferred, but not required. In this instance it would make sense, as the following line contains only two words.

Reply:
Done, thanks.